### PR TITLE
cue: 0.3.0 -> 0.3.2

### PR DESCRIPTION
**Pull request by eonpatapon**
###### Motivation for this change

New release.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/tools/cue/default.nix [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
+  version = "0.3.2";
 
   src = fetchgit {
     url = "https://cue.googlesource.com/cue";
     rev = "v${version}";
-    sha256 = "1h3809xgmn7dr57i3cnifr7r555i3zh3kfsv0gxa9nd7068w19xm";
+    sha256 = "0rfgpq4dyd3zm07vcjzn5vv0dhvvryrarxc50sd2pxagbq5cqc8l";
   };
 
   vendorSha256 = "10kvss23a8a6q26a7h1bqc3i0nskm2halsvc9wdv9zf9qsz7zjkp";
```
Comment (SuperSandro2000) [inline pkgs/development/tools/cue/default.nix:13]: Please add ``-s -m`` to buildFlagsArray.

Comment (eonpatapon) [reply pkgs/development/tools/cue/default.nix:13]: What are theses options for ?

Comment (SuperSandro2000) [reply pkgs/development/tools/cue/default.nix:13]: They strip debugging symbols.

Comment (eonpatapon) [reply pkgs/development/tools/cue/default.nix:13]: @SuperSandro2000  Seems it's `-s -w` for go, updated

**Pull request merged**